data/migrations/0001_create-profile.js

```diff
@@ -0,0 +1,20 @@
+exports.up = (knex) => {
+  return (
+    knex.schema
+      // .raw Section causes an error
+      // Error: CREATE EXTENSION IF NOT EXISTS "uuid-ossp" - SQLITE_ERROR: near "EXTENSION": syntax error
```

Comment:
I don't think we should change this table to exclude the Create Extension, this is part of the original code provided to relate to Okta, which I know you would not have any reason to know.  This line is compatible with PostgreSql so it's only using Sqlite which is causing the error.